src/mongocxx/collection.cpp

```diff
@@ -47,18 +47,6 @@
 #include <mongocxx/result/update.hpp>
 #include <mongocxx/write_concern.hpp>
 
-namespace {
-enum class cursor_flag : uint32_t {
```

Comment:
Hmm I wonder whether that's an indication that the flag was unnecessary, or that our implementation is incomplete...